docs/source/certs_management.md

```diff
@@ -379,3 +379,76 @@ Certificate:
+
+```
+fabric-ca-client enroll --<OTHER-OPTIONS>
+
+fabric-ca-client reenroll --csr.keyrequest.reusekey --mspdir <LOCATION-OF-IDENTITY'S-MSP-DIRECTORY-THAT-CONTAINS-THE-EXISTING-PRIVATE-KEY> --<OTHER-OPTIONS>
+```
+
+Replace the peer's enrollment certificate or TLS certificate.
+
+The peer's enrollment certificate is configured to be in the peer's `<peer.mspConfigPath>/signcerts` directory (if private key was not reused the new private key should also be updated in the `<peer.mspConfigPath>/keystore` directory).
+
+The peer's TLS certificate is configured to be in the peer's `<peer.tls.cert.file>` directory/file (if private key was not reused the new private key should also be updated in the `<peer.tls.key.file>`).
+
+Restart the peer.
+
+### Renew Orderer Certificates
+
+For orderer enrollment certificates you can also use the `enroll` or `reenroll` function, using the steps described above for peers.
+
+Orderer TLS certificates must be reenrolled with the option to reuse the private key. This is due to the orderer TLS certificates being configured in the system and application channels (orderer to orderer communication is verified using the public key in the configured TLS certificate). As of v1.4.9 and v2.2.1, orderer nodes verify a matching key rather than the entire configured TLS certificate, enabling orderer TLS certificate renewal without requiring a channel configuration update to update the certificate.
+
+Use `reenroll` and reuse the TLS certificate's private key by passing the `csr.keyrequest.reusekey` option and indicating the `mspdir` location of the existing private key (the private key is found in the `<mspdir>/keystore` directory):
+
+```
+fabric-ca-client reenroll --csr.keyrequest.reusekey --mspdir <LOCATION-OF-IDENTITY'S-MSP-DIRECTORY-THAT-CONTAINS-THE-EXISTING-PRIVATE-KEY> --<OTHER-OPTIONS>`
+```
+
+Replace the orderer's enrollment certificate or TLS certificate.
+
+The orderer's enrollment certificate is configured to be in the orderer's `<General.LocalMSPDir>/signcerts` directory (if private key was not reused the new private key should also be updated in the `<General.LocalMSPDir>/keystore` directory).
+
+The orderer's TLS certificate is configured to be in the orderer's `<General.TLS.Certificate>` directory/file (don't update the private key since it was reused).
+
+Restart the orderer node.
+
+### Renew Orderer TLS Certificates Without Reusing the Private Key
+
+While it is recommended to reuse the private key for orderer TLS certificate renewal, this may not be possible in all situations. Additional steps and planning are required since each of the ordering service channels (including system channel, if used) must be updated for the new orderer TLS certificate.
```

Comment:
```suggestion
While it is recommended to reuse the private key for orderer TLS certificate renewal, this may not be possible in all situations. Additional steps and planning are required since each ordering service channel (including the system channel, if used) must be updated for the new orderer TLS certificate.
```